tests/detect/detect_test.py

```diff
@@ -390,71 +405,46 @@ def test_detect_overwrite(
     def test_detect_fulfill_minimum_detection_requirements(
         self, yolov8m: Yolov8, cyclist_mp4: Path
     ) -> None:
-        deviation = 0.2
-
         class_counts = self._get_detection_counts_for(cyclist_mp4, yolov8m)
```

Comment:
Shouldn't the expected duration also be `timedelta(seconds=3)` instead `DEFAULT_EXPECTED_DURATION` like in test_detection_in_rotated_video?

Reply:
That's true, I changed it.